src/cephadm/cephadm

```diff
@@ -235,11 +257,13 @@ class NFSGanesha(object):
     }
 
     def __init__(self,
+                 ctx: CephadmContext,
                  fsid,
                  daemon_id,
                  config_json,
                  image=DEFAULT_IMAGE):
-        # type: (str, Union[int, str], Dict, str) -> None
+        # type: (CephadmContext, str, Union[int, str], Dict, str) -> None
```

Comment:
just keeping the code as close to what exists as possible. I too prefer the inline syntax as well, but if replacing the existing mixed-syntax with just one approach would be yet another huge PR -- was definitely trying to avoid that ;)

Reply:
yep. agree. it's just that mypy just doesn't want to see both and instead raises an error.

Reply:
![image](https://user-images.githubusercontent.com/2574405/103636174-e8b03100-4f49-11eb-816f-a0adf42f6f70.png)

Reply:
oh, I see what you mean... I left the `ctx: CephadmContext` there. _sigh_ it was probably muscle memory. I'll fix that.